brigade/views.py

```diff
@@ -283,5 +283,6 @@ def brigade_list():
 @app.route('/', methods=['GET'])
 def index():
     brigades = cfapi.get_brigades(official_brigades_only=True)
+    brigades_total = len(brigades)
     brigades = json.dumps(brigades)
-    return render_template("index.html", brigades=brigades)
+    return render_template("index.html", brigades=brigades, brigades_total=brigades_total)
```

Comment:
Hah, on one hand, this is awesome. On the other hand, the number is currently wrong because the CfAPI is so broken. I've been trying to fix it but it's very very flaky.